### ISSUE TITLE
Don't download nodejs/less every deploy?

### ISSUE THREAD

**Issue by rcoup** [nigma/heroku-django-cookbook#2]
Would be great if the post_compile scripts only downloaded nodejs/npm/less if they weren't already there.

**Comment by audleman** [comment]
:+1: I would love to see this

**Comment by philippTheCat** [comment]
I dont think this is possible. Afaik the slugs which heroku creates are recreated from scratch on every build.
